WeTransferPRLinter/Tests/WeTransferPRLinterTests/XCResultSummaryReporterTests.swift

```diff
@@ -67,10 +67,13 @@ final class XCResultSummartReporterTests: XCTestCase {
         WeTransferPRLinter.lint(using: danger, swiftLintExecutor: MockedSwiftLintExecutor.self, reportsPath: buildFolder.path, fileManager: stubbedFileManager, environmentVariables: [:])
 
         XCTAssertEqual(danger.messages.map(\.message), [
-            "PRLinterAppTests: Executed 10 tests, with 1 failures in 0.097 seconds"
+            "PRLinterAppTests: Executed 10 tests, with 1 failures and 1 retried tests in 0.097 seconds"
         ], "It should only report the actual failed test, instead of also the retried succeeded one")
```

Comment:
I don't feel this comment is still up-to-date.

And a question, to see if I understand this correctly: in this case, we executed 10 tests in total, of which one was a failure, and one a rerun. So that'd be running 9 unique tests. Is that assumption correct?

Reply:
The 10 includes retries, so yes, that's kind of correct. It depends on the number of retries that were needed. However, we're using the value returned by Xcode directly for the total number of tests, so it's not a calculation mistake by us if even